src/helpers/pagination.js

```diff
@@ -0,0 +1,58 @@
+/**
+ * @class
+ * @classdesc - Pagination Class
+ */
+class Pagination {
+  /**
+   * @constructor
+   * @memberof Pagination
+   * @param {number|string} page
+   * @param {number|string} limit
+   */
+  constructor(page = 1, limit = 10) {
+    this.page = Number(page, 10);
+    this.limit = Number(limit, 10);
+  }
+
+  /**
+   * @memberof Pagination
+   * @param {number} page
```

Comment:
This function takes no `param`

Reply:
Yes, you are right. I have corrected it. Thank you.